### DISCUSSION TITLE
Is it possible to have Self-Hosted runners in combination with GitHub-Hosted runners?

### DISCUSSION THREAD

**Discussion by apoorva-marisomaradhya** [actions/runner#2731] [Q&A]
Github actions has the functionality to run hosted runners for an organization (up to 50,000 minutes per month). We would like to know if it is possible to create an Actions environment that will first run until our free minutes are expired, then bring up a self-hosted environment for continued action runs after these minutes have expired.

**Comment by Obagoal1** [comment]
Follow me up
